pyftpdlib/handlers.py

```diff
@@ -2662,6 +2678,56 @@ def ftp_MDTM(self, path):
             self.respond("213 %s" % lmt)
             return path
 
+    def ftp_MFMT(self, path, timeval):
+        """ Sets the last modification time of file to timeval
+        3307 style timestamp (YYYYMMDDHHMMSS) as defined in RFC-3659.
+        On success return the modified time and file path, else None.
+        """
+        # Note: the MFMT command is not a formal RFC command
+        # but stated in the following MEMO:
+        # https://tools.ietf.org/html/draft-somers-ftp-mfxx-04
+        # this is implemented to assist with file synchronization
+
+        line = self.fs.fs2ftp(path)
+
+        if len(timeval) != len("YYYYMMDDHHMMSS"):
+            why = "Invalid time value provided"
```

Comment:
Perhaps it makes sense to give a hint about what the right format is, hence something like "Invalid time format; expected: YYYYMMDDHHMMSS"

Reply:
Thank you Giampaolo! Really appreciate your guidance. I will make the suggested changes and submit another PR.